string nodeName
int64 orderNumber
int64 carrierNumber
string hash
